### PR TITLE
Use all-in-one image

### LINKED ISSUE #2566: Update meteor-ferretdb
https://github.com/FerretDB/meteor-ferretdb

* Update to use all-in-one image, remove `postgres` and `setup` Docker Compose services
* Send PR to upstream

### PR DESCRIPTION
**Pull request by b1ron**
Closes FerretDB/FerretDB#2566.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,36 +15,16 @@ services:
     image: nginx
     hostname: 'nginx'
     container_name: 'nginx'
+    depends_on:
+      - 'ferretdb'
```
Comment (AlekSi) [inline docker-compose.yml]: It does not wait for container to be ready, so in that case it does nothing

File: docker-compose.yml [hunk 2]
```diff
@@ -15,36 +15,16 @@ services:
...
-    hostname: 'postgres'
-    container_name: 'postgres'
-    ports:
-      - 5432:5432
-    environment:
-      - POSTGRES_HOST_AUTH_METHOD=trust
-      - POSTGRES_DB=meteor
   ferretdb:
-    image: ghcr.io/ferretdb/ferretdb:latest
+    image: ghcr.io/ferretdb/all-in-one
```
Comment (AlekSi) [inline docker-compose.yml]:
```suggestion
    image: ghcr.io/ferretdb/all-in-one:latest
```

File: docker-compose.yml (hunk 1 above)
Comment (b1ron) [reply docker-compose.yml]: I will try adding a healthcheck command and send a ping to check if it's ready.

Comment (b1ron) [reply docker-compose.yml]: I used `pg_isready` instead, it seemed like the correct approach.

**Comment by b1ron** [comment]
@AlekSi PTAL.

File: docker-compose.yml (hunk 1 above)
Comment (AlekSi) [reply docker-compose.yml]: I think we should just remove it – nginx does not depend on PostgreSQL at all, and FerretDB reconnects automatically

Comment (b1ron) [reply docker-compose.yml]: I kept getting 502 errors from nginx without that healthcheck. But running `docker compose restart` after building the app resolves the issue.

So I will remove it.

Comment (b1ron) [reply docker-compose.yml]: @AlekSi PTAL.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**